app/string.js

```diff
@@ -17,7 +17,7 @@ String.prototype.ucFirst = function() {
     });
 };
 String.prototype.isQuestion = function() {
-    return /[?]/.test(this);
+    return /[?]$/.test(this);
```

Comment:
**Code Quality - Critical**

- The function should only return true if question mark is at the end of the statement